TheMovieDb/Modules/Home/HomeView.swift

```diff
@@ -7,10 +7,34 @@
 
 import UIKit
 
+enum FeedTypes: CaseIterable {
```

Comment:
What do you want to perform with CaseIterable?

Reply:
I need it to get the types array and use it on the collection delegate and data source

`
func collectionView(
        _ collectionView: UICollectionView,
        numberOfItemsInSection section: Int
    ) -> Int {
        return FeedTypes.allCases.count
    }
`